neurots/generate/algorithms/tmdgrower.py

```diff
@@ -58,24 +59,14 @@ def __init__(
         """TMD basic grower."""
         super().__init__(input_data, params, start_point, context)
         self.bif_method = bif_methods[params["branching_method"]]
-        self.params = copy.deepcopy(params)
```

Comment:
I was keeping this for safety reasons, could we keep it even though a copy might not be necessary?

Reply:
I removed it from here because it is already in the `__init__` of `AbstractAlgo`, so calling `super().__init__()` already performs the deep copy.